manifests/b/BlueJTeam/BlueJ/5.0.1/BlueJTeam.BlueJ.installer.yaml

```
@@ -1,13 +1,22 @@
-# Created using wingetcreate
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
 
 PackageIdentifier: BlueJTeam.BlueJ
 PackageVersion: 5.0.1
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
 Installers:
-- InstallerLocale: en-US
-  Architecture: x64
-  InstallerType: msi
-  InstallerUrl: https://www.bluej.org/download/files/BlueJ-windows-501.msi
-  InstallerSha256: D44B73C0C758ABC2E57CAAF6F835409A686FD37817C01C0EA319D3D3CCE9A01C
-  ProductCode: '{39F0200A-540D-43C5-A845-6D51BA794175}'
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  Scope: machine
+  InstallerUrl: https://www.bluej.org/download/files/BlueJ-windows-501.msi
+  InstallerSha256: D44B73C0C758ABC2E57CAAF6F835409A686FD37817C01C0EA319D3D3CCE9A01C
+#  ProductCode: 
+  UpgradeBehavior: install

```

```suggestion
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: msi
  Scope: machine
  InstallerUrl: https://www.bluej.org/download/files/BlueJ-windows-501.msi
  InstallerSha256: D44B73C0C758ABC2E57CAAF6F835409A686FD37817C01C0EA319D3D3CCE9A01C
  ProductCode: '{39F0200A-540D-43C5-A845-6D51BA794175}'
  InstallerSwitches:
    Custom: ALLUSERS=1
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: msi
  Scope: user
  InstallerUrl: https://www.bluej.org/download/files/BlueJ-windows-501.msi
  InstallerSha256: D44B73C0C758ABC2E57CAAF6F835409A686FD37817C01C0EA319D3D3CCE9A01C
  ProductCode: '{39F0200A-540D-43C5-A845-6D51BA794175}'
  InstallerSwitches:
    Custom: ALLUSERS=2
  UpgradeBehavior: install
```
No idea how this work, but somehow `Custom: ALLUSERS=2` don't require Administrator access